doc/releases/changelog-dev.md

```diff
@@ -348,7 +348,12 @@ of operators. [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
   (array(-0.68362956), array(0.21683382))
   ```
 
-* New FlipSign operator that flips the sign for a given basic state. [(#2780)](https://github.com/PennyLaneAI/pennylane/pull/2780)
+* Adds the `Controlled` symbolic operator to represent a controlled version of any
+  operation.
+  [(#2634)](https://github.com/PennyLaneAI/pennylane/pull/2634)
+
+* New FlipSign operator that flips the sign for a given basic state. 
```

Comment:
Code example needed

Reply:
I will do this 😄. Just going to use a slightly modified example from the one in the docs